en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md

```diff
@@ -64,7 +62,7 @@ You will be running three servers (analytics worker, analytics dashboard, and th
 
 To allow the Micro Integrator server and the Analytics server to communicate using SSL, the digital certificates should be shared by the two servers. See the instructions on [adding SSL certificates to keystores](../../../setup/security/importing_ssl_certificate).
     
-For testing, you can simply copy the keystore files from the `<EI_6.x.x_HOME>/wso2/analytics/resources/security` directory to the `MI_HOME/resources/security` directory.
+For testing, you can simply copy the keystore files from the `<EI_ANALYTICS_HOME>/wso2/analytics/resources/security` directory to the `MI_HOME/resources/security` directory.
```

Comment:
We decided to remove this line, since the keystores are now same in both packs

Reply:
removed